examples/subplots_axes_and_figures/zoom_inset_axes.py

```diff
@@ -0,0 +1,60 @@
+"""
+======================
+Zoom region inset axes
+======================
+
+Example of an inset axes and a rectangle showing where the zoom is located.
+
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def get_demo_image():
+    from matplotlib.cbook import get_sample_data
+    import numpy as np
+    f = get_sample_data("axes_grid/bivariate_normal.npy", asfileobj=False)
+    z = np.load(f)
+    # z is a numpy array of 15x15
+    return z, (-3, 4, -4, 3)
```

Comment:
Just stumbled over this. To make it clear what those magic numbers are use
~~~
extent = (-3, 4, -4, 3)
return z, extent
~~~

Reply:
Thanks, fixed, though I have a sneaky suspicion that I had it that way, and someone suggested I shorten it (ahem, @anntzer :wink:)

Reply:
For once I don't think it's my fault :) (?)